tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "preserve"
+    "jsx": "react",
```

Comment:
Is _react_ better than _preserve_?

Reply:
This change is associated with react-scripts new implementation. It's automatically made when executing 
`npm install --save --save-exact react-scripts@4.0.1` https://github.com/facebook/create-react-app/blob/master/CHANGELOG.md#401-2020-11-23

Reply:
Thanks!